jina/ndarray/sparse/scipy.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
I would move this module ndarray into proto. Or even create one data where you have ndarray and proto. But I would not have it at the same level as executors, peapods, drivers, etc...